FV3/atmos_cubed_sphere/model/fv_arrays.F90

```diff
@@ -278,7 +278,7 @@ module fv_arrays_mod
                             !< monotonicity constraint of Huynh, which is less diffusive 
                             !< than other constraints. For hydrostatic simulation, 8 
                             !< (the L04 monotonicity constraint) is recommended; for 
-                            !< nonhydrostatic simulation, the completely unlimited (“linear” 
+                            !< nonhydrostatic simulation, the completely unlimited ("linear" 
```

Comment:
I agree! I don't want to change things and cause them to look at diffs like these :/.  But these non-ascii characters break Serialbox -- this change was accepted in the fv3gfs repo previously. I looked into making Serialbox not blow up, but I feel it might be a bigger change than we want to address in that repo vs a comment here....